Counter/app/build.gradle

```diff
@@ -19,12 +19,12 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 32
```

Comment:
similar to the other pull request's comment (https://github.com/googlecodelabs/android-accessibility/pull/34#discussion_r819133076), i would recommend reverting the sdk versions to 31 for now.

Reply:
Hi Caren, thanks for the reminder. Having got the comment for the PR#34 I reverted sdk version in this PR as well but didn't push. Now it's done. Please review it and help to merge. TY!